pennylane/transforms/transpile.py

```diff
@@ -0,0 +1,156 @@
+"""
+Contains the transpiler transform.
+"""
+from typing import Union, List
+import networkx as nx
+
+from pennylane import apply
+from pennylane.ops.qubit import SWAP
+from pennylane.ops import __all__ as all_ops
+from pennylane.tape import QuantumTape
+from pennylane.transforms import qfunc_transform
+from pennylane.tape import stop_recording
+from pennylane.wires import Wires
+
+
+@qfunc_transform
+def transpile(tape: QuantumTape, coupling_map: Union[List, nx.Graph]):
+    """ Transpile a circuit according to a desired coupling map
```

Comment:
```suggestion
    """Transpile a circuit according to a desired coupling map
```
Just a minor formatting suggestion :)